Image/tests/Image-specs.js

```diff
@@ -0,0 +1,60 @@
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+
+import Image from '../Image';
+
+const src = {
+	hd: 'https://via.placeholder.com/200x200',
+	fhd: 'https://via.placeholder.com/300x300',
+	uhd: 'https://via.placeholder.com/600x600'
+};
+
+describe('Image', () => {
+	test('should select a src', () => {
+		render(<Image src={src} />);
+
+		const image = screen.getAllByRole('img')[1];
+
+		const actual = image.getAttribute('src');
+		const expected = 'https://via.placeholder.com/300x300';
+
+		expect(actual).toBe(expected);
+	});
+
+	test('should return a DOM node reference for `componentRef`', () => {
+		const ref = jest.fn();
+		render(<Image ref={ref} src={src} />);
+
+		const expected = 'DIV';
+		const actual = ref.mock.calls[0][0].nodeName;
+
+		expect(actual).toBe(expected);
+	});
+
+	test('sizing fill', () => {
```

Comment:
Could you improve the last 3 tests' description? eg. 'should render image with sizing `fill`/ `fit`/ `none`'